student-store-ui/src/components/App/App.jsx

```diff
@@ -36,7 +35,7 @@ export default function App() {
     React.useEffect(() => {
         const fetchData = async () => {
             try {
-                const response = await axios.get(API_URL);
+                const response = await axios.get("http://localhost:3001/store");
```

Comment:
You can save time and pain by storing your host as a const. It will change depending where it is deployed as well so you can do something like const HOST = 'http://localhost:3001`